dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -18,23 +18,25 @@ struct Clip: Hashable {
     let thumbnail: UIImage
     let isHighlighted: Bool
     let metadata: ClipMetadata
+    let showCode: Bool
     
-    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata) {
+    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata, showCode: Bool) {
         self.url = url
         self.thumbnail = thumbnail
         self.isHighlighted = isHighlighted
         self.metadata = metadata
+        self.showCode = showCode
```

Comment:
The addition of the `showCode` property to the `Clip` struct is a good improvement, as it allows for more control over the display of code in the clips. This can be useful for showcasing code snippets or providing additional information about the clip.